src/locales/en.json

```diff
@@ -63,9 +63,11 @@
       "transaction_count": "Transaction Count",
       "address_count": "Address Count",
       "total_dao_deposit_depositor": "Total Nervos DAO Deposit & Depositors",
-      "total_dao_deposit": "Total Nervos DAO Deposit",
-      "total_dao_depositor": "Total Depositors in History",
-      "new_dao_deposit_depositor": "Daily Deposit",
+      "total_dao_deposit_title": "Total Deposit",
+      "total_dao_deposit": "Total Deposit",
+      "total_dao_depositor": "Total Depositors",
+      "new_dao_deposit_depositor": "Daily Nervos DAO Deposit",
```

Comment:
why key is `xxx_depositor` but the value is `xxx Deposit`

Reply:
`new_dao_deposit_depositor` is full name for this chart, but the value of `new_dao_deposit_depositor ` is decided by product manager.

Reply:
ok, but I think `depositor` and `deposit` are completely different concepts

Reply:
Yeah, you are right. But this name is daily deposit and despositor chart title, and I think the combine of deposit and depositor can clearly display the meaning of this chart

Reply:
ok, got it, but if this is a title, would it be better to add `_title` to the end like other titles

Reply:
We have two titles for chart which includes the title of `chart group page` and the title of chart detail page.

`_title` is for `chart group page` and the combination is for chart detail page

Reply:
Maybe the names of charts are a litter confusion and I will refactor them in another pr later